central/alert/datastore/internal/search/searcher_impl.go

```diff
@@ -140,29 +151,17 @@ func formatSearcher(unsafeSearcher blevesearch.UnsafeSearcher) search.Searcher {
 	var filteredSearcher search.Searcher
 	if env.PostgresDatastoreEnabled.BooleanSetting() {
 		// Make the UnsafeSearcher safe.
-		filteredSearcher = alertPosgresSACSearchHelper.FilteredSearcher(unsafeSearcher)
+		filteredSearcher = alertPostgresSACSearchHelper.FilteredSearcher(unsafeSearcher)
 	} else {
 		filteredSearcher = alertSearchHelper.FilteredSearcher(unsafeSearcher) // Make the UnsafeSearcher safe.
+		filteredSearcher = sortfields.TransformSortFields(filteredSearcher, mappings.OptionsMap)
+		filteredSearcher = paginated.Paginated(filteredSearcher)
 	}
-	transformedSortFieldSearcher := sortfields.TransformSortFields(filteredSearcher, mappings.OptionsMap)
-	paginatedSearcher := paginated.Paginated(transformedSortFieldSearcher)
-	defaultSortedSearcher := paginated.WithDefaultSortOption(paginatedSearcher, defaultSortOption)
-	withDefaultViolationState := withDefaultActiveViolations(defaultSortedSearcher)
+	withDefaultViolationState := withDefaultActiveViolations(filteredSearcher)
```

Comment:
Is `applyDefaultState` same as this? If so, we can drop it off from one of the places.